### ISSUE TITLE
Wrong output with classes with template string and ternary operator

### ISSUE THREAD

**Issue by gyandeeps** [MithrilJS/mopt#59]
Version: 5.0.2

code: 

```js
m(
    "p",
    {
        className: `mithril-Pager-page-arrow ${vnode.attrs.currentPage > 1 ? "" : "mithril-Pager-page-arrow--disabled"}`
    },
    "test"
)
```

In the output `className` is not included.

For repo: https://github.com/gyandeeps/mithril-hn
File: https://github.com/gyandeeps/mithril-hn/blob/82879b862da659491d441837f9953f40337ed033/src/js/components/Pager.js

Notes:

* If I don't use template string then it works.
`className: "mithril-Pager-page-arrow " + (vnode.attrs.currentPage > 1 ? "" : "mithril-Pager-page-arrow--disabled")`

**Comment by tivac** [comment]
Interesting, it seems like I should already have a test for that. I'll look into it!

**Comment by tivac** [comment]
This is fixed in `mopt@5.1.1`, thanks again for the report & PR @gyandeeps!
